packages/scripts.nix

```
@@ -74,7 +74,7 @@
     text = ''
       exitcode=0
 
-      tags="${lib.concatStringsSep "," pkgs.contrast.tags}"
+      tags="${lib.concatStringsSep "," pkgs.contrastPkgs.contrast.tags}"
```

```suggestion
      tags="${lib.concatStringsSep "," contrastPkgs.contrast.tags}"
```

(same for the other instances of `pkgs.contrastPkgs`)